--- requests-2.33.1/src/requests/certs.py	2026-04-24 15:42:47.921673130 +0200
+++ requests-2.33.1/src/requests/certs.py	2026-04-24 15:42:47.971459284 +0200
@@ -12,7 +12,13 @@
 packaged CA bundle.
 """

-from certifi import where
+import os
+
+if os.path.exists("/etc/certs/ca-certificates.crt"):
+    def where():
+        return "/etc/certs/ca-certificates.crt"
+else:
+    from certifi import where

 if __name__ == "__main__":
     print(where())
